src/cli.js

```diff
@@ -26,15 +26,57 @@
 *
 */
 
-import transformCallback from './transform';
-import createValidator from './validate';
+import transform from './transform';
+import validate from './validate';
 import {Transformer} from '@natlibfi/melinda-record-import-commons';
+import moment from 'moment';
 
 const {runCLI} = Transformer;
 
 run();
 
 async function run() {
-	const validateCallback = await createValidator();
-	runCLI({name: 'melinda-record-import-transformer-helmet', transformCallback, validateCallback});
+	const transformerSettings = {
+		name: 'melinda-record-import-transformer-helmet',
+		yargs: [
+			{option: 'v', conf: {alias: 'validate', default: false, type: 'boolean', describe: 'Validate records'}},
+			{option: 'f', conf: {alias: 'fix', default: false, type: 'boolean', describe: 'Validate & fix records'}}
+		],
+		callback: startTransform
+	};
+	runCLI(transformerSettings);
+
+	async function startTransform(stream, args, spinner, handleRecordsOutput) {
```

Comment:
Use a single object argument instead of positionals when there are many args (Otherwise the order matters and that can lead to trouble):
```js
async function startTransform({stream, args, spinner, handleRecordsOutput}) {
```
(Object destructuring gives you those variables without having to access the object)